contracts/CoveragePool.sol

```diff
@@ -223,6 +224,26 @@ contract CoveragePool is Ownable {
         assetPool.claim(recipient, amountToSeize(portionToSeize));
     }
 
+    /// @notice Seizes funds from the coverage pool and puts them aside for the
+    ///         recipient to withdraw. This function in contrary to the other
+    ///         'seizeFunds' accepts a specific amount to be seized from the pool
+    ///         but no more than the pool holds.
+    /// @dev    Note the order of params compared to the other 'seizeFunds'.
+    ///         Here we seize a specific amount of funds.
+    /// @param  amount Amount to be seized from the pool.
+    /// @param  recipient Address that will receive the pool's seized funds.
+    function seizeFunds(uint256 amount, address recipient)
```

Comment:
To lower the possibility of a human error we could have the same order of params but use different names for functions:
- `seizePortionOfPool(address recipient, uint256 portionToSeize)`
- `seizeAmountFromPool(address recipient, uint256 amount)`